string type
Head_Goal goal
---
string result
